### ISSUE TITLE
NTR: Asparagine low in CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8419]
New term request Asparagine low in CSF (elthat_181228121439):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000408
- label: Reduced CSF asparagine concentration
- parents: [Abnormal CSF aspartate family amino acid concentration (HP:0500205)](https://hpo.jax.org/app/browse/term/HP:0500205)
- pmid: 
- def: The concentration of asparagine in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8419
- synonyms:
